src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleanedStyles = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter(Boolean);
+  const result = {};
+
+  cleanedStyles.forEach((style) => {
+    const colonIndex = style.indexOf(':');
+
+    if (colonIndex === -1) {
+      return;
+    }
+
+    const property = style.slice(0, colonIndex).trim();
+    const value = style.slice(colonIndex + 1).trimStart();
```

Comment:
Consider using `trim()` instead of `trimStart()` to remove both leading and trailing spaces from the value. This ensures that any extra spaces at the end of the value are also removed.